.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -0,0 +1,71 @@
+name: Bug Report
+description: Report broken functionality within Radius
+title: "<BUG TITLE>"
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "## Bug information"
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: How can we recreate this bug? Be specific.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Observed behavior
+      description: What you're experiencing that you believe is a bug.
+      placeholder: |
+        If applicable, add screenshots to help explain your problem.
+
+        Tip: You can attach images or files by clicking this area to highlight it and then dragging files in.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Desired behavior
+      description: What you're expecting to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Workaround
+      description: Have you found a workaround to get you unblocked?
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: "## System information"
+  - type: input
```

`rad version` will have a multi-line output. A `textarea` input would probably be better here.
```suggestion
  - type: textarea
```

---

Oh yes, that's right.